### PR TITLE
First version of profiling docs with a short guide

### PR DESCRIPTION
**Pull request by jernejstrasner**
This is the first version of the profiling docs. Some of the content has been moved from documents that were previously shared with beta users. This content helps users setup the SDKs for the beta version of profiling.

### CODE REVIEW

**Comment by jernejstrasner** [comment]
This addresses issues https://github.com/getsentry/team-profiling/issues/40 and https://github.com/getsentry/team-profiling/issues/39

File: src/platforms/common/profiling/index.mdx [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+
+## Enabled Profiling
+
+<PlatformSection supported={["apple"]}>
+
+```swift
+SentrySDK.start { options in
+    options.dsn = "..."
+		options.tracesSampleRate = 1.0
+    options.enableProfiling = true
```
Comment (jernejstrasner) [inline src/platforms/common/profiling/index.mdx:85]: @indragiek we now have `profilingSampleRate` instead right?

**Review by imatwawana — COMMENTED** [review]
This mostly looks good! I've made a number of fixes to standardize the styling, removed the [beta] from the page titles and the navigation since we don't typically do that for new features, and made some other wording suggestions. Ping me if anything isn't clear. I'd like to take another look at this before it's merged.

File: src/platforms/common/profiling/index.mdx [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+
+<Note>
+
+Profiling is still an experimental, work-in-progress feature, and its API may be unstable. It may also have bugs. Feedback is appreciated, and should be submitted as an issue in the repository linked on the right side of this page.
+
+</Note>
+
+<Note>
+
+If you don't have access to Profiling please reach out to [profiling@sentry.io](mailto:profiling@sentry.io).
```
[the remark this answers is not in the document]
Comment (jernejstrasner) [reply src/platforms/common/profiling/index.mdx:41]: Yeah the sentry dashboard.
I feel the language by adding the `at` is weird now. Usually you're *reaching out to*, no? Alternatively *reach us at*?

**Comment by jernejstrasner** [comment]
@imatwawana thanks for the review! I also synced the *beta* note on all pages. Lmk how this looks now.

File: src/platforms/common/profiling/index.mdx (hunk 2 above)
[the remark this answers is not in the document]
Comment (imatwawana) [reply src/platforms/common/profiling/index.mdx:41]: I think the change you've made here (using _contact us at_) sounds good!

[the remark this answers is not in the document]
Comment (imatwawana) [reply src/platforms/common/profiling/index.mdx:41]: What about "...reach out to us at profiling@sentry.io

File: src/platforms/common/profiling/index.mdx [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+
+<Note>
+
+Profiling is currently in beta. Beta features are still in-progress and may have bugs. We recognize the irony. For access, contact us at [profiling@sentry.io](mailto:profiling@sentry.io).
+
+</Note>
+
+<Note>
+
+If you don't have access to Profiling please reach out to [profiling@sentry.io](mailto:profiling@sentry.io).
```
Comment (imatwawana) [inline src/platforms/common/profiling/index.mdx:41]: Wondering why we need the second note at all actually. "Access" just means the ability for the customer to see profiling data in sentry.io, right? 

```suggestion
<Note>

Profiling is currently in beta. Beta features are still in-progress and may have bugs. We recognize the irony. For access, contact us at [profiling@sentry.io](mailto:profiling@sentry.io). 
```

File: src/platforms/common/profiling/index.mdx [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+    options.dsn = "..."
+    options.tracesSampleRate = 1.0
+}
+```
+
+</PlatformSection>
+
+<PlatformSection supported={["android"]}>
+
+In `AndroidManifest.xml`:
```
Comment (imatwawana) [inline src/platforms/common/profiling/index.mdx:64]: The user is already on an Android page, and we have two sentences before the code snippet ending in a colon. We don't do this on the Apple version of these docs. Can we remove this or do we need this information here?
```suggestion
```

File: src/platforms/common/profiling/index.mdx [hunk 5]
```diff
@@ -0,0 +1,124 @@
...
+
+The <PlatformIdentifier name="profiles-sample-rate" /> setting is *relative* to the <PlatformIdentifier name="traces-sample-rate" /> setting.
+
+</Note>
+
+</PlatformSection>
+
+<PlatformSection supported={["android"]}>
+
+In `AndroidManifest.xml`:
```
Comment (imatwawana) [inline src/platforms/common/profiling/index.mdx:99]:
```suggestion
```

File: src/docs/product/profiling/index.mdx [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+
+![Profiling dashboard displaying a flamegraph](profiling-dashboard.png)
+
+The typical approach to measuring app performance involves setting up automated performance tests for various user flows that run during continuous integration (CI), or manually profiling hot paths in your app using tools like Apple Instruments or Android Profiler. These are excellent tools, but they have numerous drawbacks:
+
+- **The environment is near-optimal and highly controlled.** Developers often use the latest, most powerful devices, and are testing apps on excellent network conditions. These conditions do not always reflect real world performance scenarios — users may be running your app on low-end hardware, older operating system versions, or poor network conditions. These conditions can be simulated to some extent, but the complete matrix of variables is too large to be able to test every case on a regular basis.
+- **Only a subset of user flows are being tested.** Performance tests are typically set up for the most common user flows, but it's impractical to build and maintain tests for every possible way in which someone may be using your app.
```
Comment (imatwawana) [inline src/docs/product/profiling/index.mdx:22]:
```suggestion
- **Only a subset of user flows are being tested.** Performance tests are typically set up for the most common user flows, but it's impractical to build and maintain tests for every possible way someone might be using your app.
```

File: src/docs/product/profiling/index.mdx [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+- **Metrics are calculated based on a small number of samples.** Performance metrics are often calculated by aggregating the results from a small number of runs. While this can be sufficient in some cases, it may not be enough data to give you an accurate, statistically significant measurement, especially if you need to account for different hardware, thermal conditions, and so on.
+- **The process is difficult and time consuming.** Manually conducting extensive performance tests on your app before every release is time consuming. Building and maintaining automated performance tests is also time consuming. Interpreting the results can be difficult and frustrating for someone who is not a performance specialist and is unfamiliar with the tools or the process.
```
Comment (imatwawana) [inline src/docs/product/profiling/index.mdx:24]:
```suggestion
- **The process is difficult and time consuming.** Manually conducting extensive performance tests on your app before every release is time consuming. Building and maintaining automated performance tests is also time consuming. Interpreting the results can be difficult and frustrating for someone who's not a performance specialist and is unfamiliar with the tools or the process.
```

File: src/docs/product/profiling/index.mdx [hunk 8]
```diff
@@ -0,0 +1,36 @@
...
+
+Sentry Profiling is designed to address all of these issues:
```
Comment (imatwawana) [inline src/docs/product/profiling/index.mdx:26]:
```suggestion
Profiling is designed to address all of these issues:
```

File: src/docs/product/profiling/index.mdx [hunk 9]
```diff
@@ -0,0 +1,36 @@
...
+
+- **The environments are diverse, and reflect real world conditions.** We collect performance data automatically from every environment that your app is running on in the real world, accounting for different device hardware, OS versions, app versions, network conditions, and more.
```
Comment (imatwawana) [inline src/docs/product/profiling/index.mdx:28]:
```suggestion
- **The environments are diverse and reflect real world conditions.** We collect performance data automatically from every environment that your app is running on in the real world, accounting for different device hardware, OS versions, app versions, network conditions, and more.
```

File: src/docs/product/profiling/index.mdx [hunk 10]
```diff
@@ -0,0 +1,36 @@
...
+- **We cover the widest possible set of user flows.** Since the data is collected from actual users using your app as they normally would, we can capture all kinds of usage patterns that you may not have thought about when building your own performance tests.
+- **Metrics are calculated using a large number of samples.** We can collect data from up to 100% of your user population in production, allowing us to compute more accurate metrics.
+- **The process is simple.** Integrate the Sentry SDK, enable profiling, ship your app, and let us worry about analyzing performance and detecting regressions between versions. We encourrage *testing in production*, which improves developer velocity and reduces the overhead of having to maintain your own tests and tooling. Our dashboard is designed to be approachable to all engineers, and not just people who have a performance engineering background.
+- **The overhead is minimal.** Depending on the platform, the overhead can be lower than 1% of your app’s CPU time.
+
+## Larn More
```
Comment (imatwawana) [inline src/docs/product/profiling/index.mdx:34]:
```suggestion
## Learn More
```

File: src/docs/product/profiling/getting-started.mdx [hunk 11]
```diff
@@ -0,0 +1,28 @@
...
+
+</Note>
+
+To set up the profiling integration, use the links for supported SDKs below. Profiling helps you see code-level profiling information for your Sentry apps.
+
+## Supported SDKs
+
+<Alert>
+
+Profiling depends on Sentry's performance monitoring product being enabled beforehand. To enable performance monitoring in the SDK check out our [Performance Monitoring guide](/product/performance/getting-started/).
```
Comment (imatwawana) [inline src/docs/product/profiling/getting-started.mdx:22]:
```suggestion
Profiling depends on Sentry's performance monitoring product being enabled beforehand. To enable performance monitoring in the SDK, check out our [Performance Monitoring guide](/product/performance/getting-started/).
```

**Review by imatwawana — APPROVED** [review]
Caught a typo and found made some small wording suggestions I missed the first time. Also made a suggestion for that outstanding note.

File: src/platforms/common/profiling/index.mdx (hunk 3 above)
Comment (jernejstrasner) [reply src/platforms/common/profiling/index.mdx:41]: Add yes. Good catch.

**Pull request merged**